packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -863,10 +868,22 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
     return <span className={this._classNames.dropdownOptionText}>{item.text}</span>;
   };
 
-  /** Render custom label for drop down item */
-  private _onRenderItemLabel = (item: IDropdownOption): JSX.Element | null => {
-    const { onRenderOption = this._onRenderOption } = this.props;
-    return onRenderOption(item, this._onRenderOption);
+  /*
+   * Render content of a multiselect item label.
+   * Text within the label is aria-hidden, to prevent duplicate input/label exposure
+   */
+  private _onRenderMultiselectOption = (item: IDropdownOption): JSX.Element => {
+    return (
+      <span id={item.id} aria-hidden="true" className={this._classNames.dropdownOptionText}>
+        {item.text}
+      </span>
+    );
+  };
+
+  /** Render custom label for multiselect checkbox items */
+  private _onRenderItemLabel = (item: IDropdownOption, id: string): JSX.Element | null => {
```

Comment:
It looks like you're spreading the `id` into the `item` object on line 854. Should this be:
```suggestion
  private _onRenderItemLabel = (item: IDropdownOption): JSX.Element | null => {
```

Reply:
ah, yup! I missed removing that when I changed the implementation 🙈